packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-## v3.3.0-extensibility-preview.4 (Feb 12, 2025)
+## v4.0.0-dev (Mar 07, 2025)
+- Upgrade `@tanstack/react-query` to v5 [#2303](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2303)
+- Refactor all API hooks to use the new `createUseQuery` and `createUseMutation` hook [#2353](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2353)
```

Comment:
Are we planning on generating some more docs around migration guide for v4 to v5?

Reply:
The plan is to refer to react-query documentation